services/arrow-flight/src/main/java/com/dremio/service/flight/DremioFlightProducer.java

```diff
@@ -113,4 +157,36 @@ public void listActions(CallContext callContext, StreamListener<ActionType> stre
   private CallHeaders retrieveHeadersFromCallContext(CallContext callContext) {
     return callContext.getMiddleware(FlightConstants.HEADER_KEY).headers();
   }
+
+  // TODO: Add this to FlightSqlUtils
```

Comment:
This should be done before I'd consider approving...

Reply:
Fixed - https://github.com/rafael-telles/arrow-experimental/pull/74/files